libs/Microsoft.MixedReality.WebRTC/Interop/AudioReadStreamInterop.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.MixedReality.WebRTC.Interop
+{
+    internal class AudioReadStreamInterop
+    {
+        [DllImport(Utils.dllPath, CallingConvention = CallingConvention.StdCall, CharSet = CharSet.Ansi,
+            EntryPoint = "mrsAudioReadStreamCreate")]
+        public static extern uint Create(PeerConnectionHandle peerHandle, int bufferMs, ref IntPtr audioReadStream);
```

Comment:
@djee-ms is this the correct pattern? Accept a safehandle & the native method has void*?

Reply:
Yes. But you probably want to mark `audioReadStream` as `out`, not `ref`, no? It's the returned handle to the newly allocated object, isn't it? So it's not valid until returned.